pandas/core/dtypes/missing.py

```diff
@@ -108,7 +109,7 @@ def _isna_new(obj):
     elif isinstance(obj, ABCMultiIndex):
         raise NotImplementedError("isna is not defined for MultiIndex")
     elif isinstance(obj, (ABCSeries, np.ndarray, ABCIndexClass,
-                          ABCExtensionArray)):
+                          ABCExtensionArray, ABCDatetimeArray)):
```

Comment:
why is this needed? is ABCDatetimeArray NOT an extension array?

Reply:
not until #24024